packages/p2p/src/components/orders/order-table/order-table-row.jsx

```diff
@@ -1,25 +1,46 @@
-import { Table } from '@deriv/components';
+import { Table, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { observer } from 'mobx-react-lite';
-import { localize, Localize } from 'Components/i18next';
+import { localize } from 'Components/i18next';
 import { secondsToTimer } from 'Utils/date-time';
 import { createExtendedOrderDetails } from 'Utils/orders';
 import ServerTime from 'Utils/server-time';
 import { useStores } from 'Stores';
 
-const OrderRow = observer(({ row: order }) => {
+const Title = ({
+    is_buy_order_type_for_user,
+    transaction_amount,
+    offer_amount,
+    order_purchase_datetime,
+    order_type,
+}) => {
+    const send_amount = is_buy_order_type_for_user ? transaction_amount : offer_amount;
+    return (
+        <React.Fragment>
+            <Text size='sm' color='prominent' line_height='xxs' weight='bold' as='p'>
+                {order_type} {send_amount}
+            </Text>
+            <Text color='less-prominent' as='p' line_height='xxs' size='xxs' align='left'>
```

Comment:
Is `align='left'` required?

Reply:
yes it's required, we have some styles which we could not remove